### PR TITLE
Add sanity check on adding a new rule property

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**

### CODE REVIEW

File: sonar-plugin/css/src/test/java/org/sonar/css/rules/CssRuleTest.java [hunk 1]
```diff
@@ -64,6 +67,23 @@ void rules_default_is_empty()
     }
   }
 
+  /*
+   * This test raises awareness of the consequence of a rule adding or removing a rule property.
+   * If a new rule property is added, we should inform the SonarCloud team about it on release.
```
Comment (ilia-kebets-sonarsource) [inline sonar-plugin/css/src/test/java/org/sonar/css/rules/CssRuleTest.java:72]: maybe link to some process on how to contact them?

Comment (yassin-kammoun-sonarsource) [reply sonar-plugin/css/src/test/java/org/sonar/css/rules/CssRuleTest.java:72]: There are none. That should be email communication in my opinion.

Comment (yassin-kammoun-sonarsource) [reply sonar-plugin/css/src/test/java/org/sonar/css/rules/CssRuleTest.java:72]: And I also mentioned this in our release procedures.

**Review by ilia-kebets-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**